R/readAxivity.R

```diff
@@ -489,9 +491,8 @@ readAxivity = function(filename, start = 0, end = 0, progressBar = FALSE, desire
       rawTime[1] = prevStart - 0.00001
       rawPos = 2
     }
-    
-    
-    frequency_observed = length(rawTime[1:rawLast]) / diff(range(rawTime[1:rawLast]))
+
+    frequency_observed = rawLast / (rawTime[rawLast] - rawTime[1])
```

Comment:
diff(range()) construct was expensive, caused most of the time increase of the PR. Can we get away with (rawTime[rawLast] - rawTime[1]) ? Or do we need to worry about non-sequential timestamps?

Reply:
Agree, we can do it this way. Non-sequential is not a concern in your solution. Non-incremental would be a theoretical concern, but I doubt this can be expected.